pyvista/utilities/geometric_objects.py

```diff
@@ -1218,6 +1219,72 @@ def Circle(radius=0.5, resolution=100):
     return pyvista.wrap(pyvista.PolyData(points, cells))
 
 
+def Superquadric(center=(0., 0., 0.), scale=(1., 1., 1.), size=0.5,
+                 thetaroundness=1., phiroundness=1.,
+                 thetaresolution=16, phiresolution=16,
+                 toroidal=False, thickness=1/3):
+    """Create an arrow.
+
+    Parameters
+    ----------
+    center : iterable, optional
+        Center of the superquadric in ``[x, y, z]``.
+
+    scale : iterable, optional
+        Scale factors of the superquadric in ``[x, y, z]``.
+
+    size : float, optional
+        Superquadric isotropic size.
+
+    thetaroundness : float, optional
+        Superquadric east/west roundness.
+        Values range from 0 (rectangular) to 1 (circular) to higher orders.
+
+    phiroundness : float, optional
+        Superquadric north/south roundness.
+        Values range from 0 (rectangular) to 1 (circular) to higher orders.
+
+    thetaresolution : int, optional
+        Number of points in the longitude direction.
+        Values are rounded to nearest multiple of 4.
+
+    phiresolution : int, optional
+        Number of points in the latitude direction.
+        Values are rounded to nearest multiple of 8.
```

Comment:
I suggest that we stick to the underscore naming convention for multi-word kwargs, see e.g. https://github.com/pyvista/pyvista/blob/844ad616f743d305ed0d635e1e7bb0106e691e7d/pyvista/utilities/geometric_objects.py#L271-L304